### ISSUE TITLE
CCVideoPlayer: support Portrait & Portrait Upside-Down

### ISSUE THREAD

**Issue by psineur** [cocos2d/cocos2d-iphone-extensions#13]
Currently only Landscape Left & Landscape Right are supported.

**Comment by psineur** [comment]
Extensions Test now rotates to any orientation in develop branch.
It's time to close this issue.
Moved to 0.2 milestone.

**Comment by alexszilagyi** [comment]
@psineur Can you tell me if it supports real-time update of the device orientation?

**Comment by psineur** [comment]
Yes, but only for the landscape orientations. 

There's a method for this declared in CCVideoPlayer.h

07.06.2012, в 15:48, alexszilagyireply@reply.github.com написал(а):

**Comment by alexszilagyi** [comment]
I know that there's a class method called [updateOrientationWithOrientation]. but its possible to do it as YouTube application does when you change the orientation of the iPad ?

**Comment by psineur** [comment]
Yes, through that method. You should call it from your app delegate method or notification, when orientation changes. 

Отправлено с iPhone

07.06.2012, в 19:51, alexszilagyireply@reply.github.com написал(а):
